bundles/org.eclipse.passage.lic.base/src/org/eclipse/passage/lic/base/io/FileCollection.java

```diff
@@ -49,9 +49,9 @@ public Collection<Path> get() throws LicensingException {
 		}
 	}
 
-	private Stream<Path> filesIn(Path path) throws IOException {
-		return Files.walk(path) //
-				.filter(Files::isRegularFile);
+	static Stream<Path> filesIn(Path path) throws IOException {
```

Comment:
`static` methods are prohibited in Passage project. This one also must stay private.

Also please rename te method itself from `filesIn` to `files`.  Tailing prepositions hardy used in the project.

Reply:
Reverted that part.